src/apis/user/user.service.ts

```diff
@@ -85,23 +86,32 @@ export class UserService {
     const checkNickName = await this.userRepository.findOne({
       where: { nickname },
     });
-    const checkPhone = await this.userRepository.findOne({
-      where: { phoneNumber },
-    });
+    // const checkPhone = await this.userRepository.findOne({
+    //   where: { phoneNumber },
+    // });
     const checkEmail = await this.userRepository.findOne({
       where: { email },
     });
     if (checkNickName) {
-      throw new NotFoundException('이미 사용 중인 닉네임 입니다.');
+      throw new ConflictException('중복된 닉네임이 존재합니다.');
     } else if (checkEmail)
```

Comment:
```suggestion
    } if (checkEmail)
```
당장 필요한건 아니지만 이전 if 문에서 return 이나 throw 가 있다면 else 문 이 필요하지 않습니다.
그리고 줄바꿈이 생긴다면 중괄호로 묶는것도 좋을듯해요

Reply:
넵 반영하겠습니다.